apps/condo/domains/organization/integrations/sbbol/utils/changeClientSecret.js

```diff
@@ -2,15 +2,22 @@ const querystring = require('querystring')
 const dayjs = require('dayjs')
 const conf = require('@open-condo/config')
 const { SbbolRequestApi } = require('../SbbolRequestApi')
+const { SBBOL_IMPORT_NAME } = require('@condo/domains/organization/integrations/sbbol/constants')
 const { getOrganizationAccessToken } = require('./getOrganizationAccessToken')
 const { getSbbolSecretStorage } = require('./getSbbolSecretStorage')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { User } = require('@condo/domains/user/utils/serverSchema')
 
 const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
 const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
 
-async function changeClientSecret ({ clientId, currentClientSecret, newClientSecret }) {
-    const accessToken = await getOrganizationAccessToken()
+async function changeClientSecret ({ clientId, currentClientSecret, newClientSecret, userId }) {
     const sbbolSecretStorage = getSbbolSecretStorage()
+    const { keystone: userContext } = await getSchemaCtx('User')
+
+    // SBBOL API allows to use any `accessToken` to change `clientSecret` because it is not bound to specific user
+    const user = await User.getOne(userContext, { importRemoteSystem: SBBOL_IMPORT_NAME, deletedAt: null })
+    const accessToken = userId ? await getOrganizationAccessToken(userId) : await getOrganizationAccessToken(user.id)
```

Comment:
[Logs with errors](https://kibana-logs.in.doma.ai/app/discover#/doc/e88dc5f0-fdeb-11ec-8627-212f76435941/production-condo-worker-2022.12.21?id=GFAxMoUBfz4a3eoYzLny) on production has shown, that this will not work, because when trying to change Client Secret using `accessToken` of a user, not belonging to our organization at SBBOL.

We should fetch only user, that belongs to our organization.
Our organization can be fetched by `SBBOL_FINTECH_CONFIG.service_organization_hashOrgId`
Take first user of that organization.


```suggestion
    const organization = await Organization.getOne(userContext, { importId: SBBOL_FINTECH_CONFIG.service_organization_hashOrgId } )
    if (!organization) {
        throw new Error('Could not fetch SBBOL service organization by importId with corresponds to SBBOL_FINTECH_CONFIG.service_organization_hashOrgId')
    }
    const user = await User.getOne(userContext, { organization: { id: organization.id }, deletedAt: null })
    if (!user) {
        throw new Error('Could not fetch User from SBBOL service organization')
    }
    const accessToken = await getOrganizationAccessToken(user.id)
```